metrics/web/lib/common/presentation/metrics_theme/state/theme_notifier.dart

```diff
@@ -14,4 +21,10 @@ class ThemeNotifier extends ChangeNotifier {
     _isDark = !_isDark;
     notifyListeners();
   }
+
+  /// Sets the [_isDark] value according to the given [isDark] value.
+  void setTheme({bool isDark}) {
+    _isDark = isDark;
```

Comment:
As I remember we should change the theme mode to the OS selected only on the Login page. But I'm wondering should we handle the following scenario: 
1. A user logs in to the Metrics with the light theme on the Login page (*OS: light*, *app: light*).
2. A user changes the theme mode to the dark on the user menu (*OS: light*, *app: dark*).
3. A user logs out of the application and is moved to the Login page  (*OS: light*, *app: ???*).

What the mode should be selected instead of `???`. According to the implementation in this PR, the mode will be light as the OS states. 

cc @illia-romanenko

Reply:
I think light is correct, we decided for the login page to follow the OS theme.

Reply:
Cool, thanks!